api/src/routes/adminRoutes.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+import * as controller from '../controllers/admin/index';
+import middleware from '../controllers/middleware';
+import Scope from '../types/scopes';
+
+export const router = express.Router();
+
+router.get('/admin/allUsers', middleware.authorize(Scope.ReadAllUsersRoles), controller.getAllUsersInfo);
```

Comment:
Yep this is all you should need for authorization. Make sure you add the role to all the environments when this merges.